simue/handler.go

```diff
@@ -448,15 +448,35 @@ func HandleProcedure(ue *simuectx.SimUe) {
 		msg := &common.UeMessage{}
 		msg.Event = common.PDU_SESS_REL_REQUEST_EVENT
 		SendToRealUe(ue, msg)
-	case common.USER_DATA_PKT_GENERATION_PROCEDURE:
+	case common.UDP_DATA_PKT_GENERATION_PROCEDURE:
+		ue.Log.Infoln("Initiating Udp Data Packet Generation Procedure")
+		msg := &common.UeMessage{}
+		msg.UserDataPktCount = ue.ProfileCtx.DataPktCount
+		if ue.ProfileCtx.DefaultAs == "" {
+			ue.ProfileCtx.DefaultAs = "192.168.250.1" // default destination for AIAB
+		}
+		msg.DefaultAs = ue.ProfileCtx.DefaultAs
+		msg.DefaultAsSrcPort = ue.ProfileCtx.DefaultAsSrcPort
```

Comment:
Should not this also be having some default UDP port ?

Reply:
done